### DISCUSSION TITLE
Feature request new user interface: expand code for new button 'Update meeting info'

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#1184] [General]
Today I upgraded my website to the new user interface.
I like how it looks and find it a lot faster than the old UI. So hurray for that!

I have one feature request that is pretty essential to me.

I am not in love with the new 'Update meeting info' button. I find it too bare bones to be helpful in the way it was before.
I have to click the URL in the footer to see the relevant meeting's details, front or back. This adds to my workload...

This email with notifications typically also goes to our Group Records Coordinator, and considering that this is usually not a technical person, these pagelinks will be daunting to he

And to be honest: what was visible in the body of the auto generated email was very efficient and just what she (and I) needed to process notifications. Now, an extra step has been added that could potentially be more confusing than helpful...

Is there any chance of bringing back the information that was there in the old format? at the very least the meeting notes?

Why the meeting notes? They contain the ID that is unique for all of my 300+ groups/meetings.
The name of many groups are similar or even identical (even within one district) and the ID is the only thing that I can use to identify them quickly and correctly. So, if the meeting notes could be included in that email format, that would really make a big difference. More efficient too. 

I realise that there is another issue here as well.
Previously, the email sent from the website page came in with the website page as the sender.
Now, it looks as if it will come in from the email address of the person entering that change request.
That makes it really hard to stay organised in our email boxes. We used to have everything from the website neatly placed in one folder. But when it comes in from individual members, that level of organisation no longer is possible.

Are there more comments on this feature? If I am the only one with problems, I may have to revert back to the old UI and miss out on all the other good stuff that was developed :(

Thanks for. your consideration!
Nicky D
al-anon,alateen.on.ca

**Comment by anchovie91471** [comment]
TSML UI doesn't offer the same feedback form as the Legacy UI, but you can make the button link to a form you create (with a forms plugin or code). Here is our FAQ link (https://wordpress.org/plugins/12-step-meeting-list/#can%20i%20set%20a%20custom%20%22update%20meeting%20info%22%20button%20in%20tsml%20ui%3F). Once you create the form, you use the code supplied in the FAQ to force the button to link to that form.
